create_tables.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS logs CASCADE;
+DROP TABLE IF EXISTS stats CASCADE;
+DROP TABLE IF EXISTS workers CASCADE;
+DROP TABLE IF EXISTS components CASCADE;
+DROP TABLE IF EXISTS users CASCADE;
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY,
+  email VARCHAR(100) NOT NULL,
+  github_token VARCHAR(100) NOT NULL,
+  github_username VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE components (
+  component_id SERIAL PRIMARY KEY,
+  user_id SERIAL REFERENCES users(user_id),
+  github_repo VARCHAR(100) NOT NULL,
+  deployment_status VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE workers (
+  worker_id SERIAL PRIMARY KEY,
+  worker_name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE stats (
+  stat_id SERIAL PRIMARY KEY,
+  worker_id SERIAL REFERENCES workers(worker_id),
+  component_id SERIAL REFERENCES components(component_id),
+  received_time TIMESTAMP NOT NULL,
```

Comment:
`timestamptz` is better

Reply:
why?

Reply:
https://medium.com/building-the-system/how-to-store-dates-and-times-in-postgresql-269bda8d6403